### ISSUE TITLE
Php::echo()

### ISSUE THREAD

**Issue by valmat** [CopernicaMarketingSoftware/PHP-CPP#51]
I have already mentioned this feature.
The fact that php uses its own output stream for printing text.
`std::cout` sends data to standard output stream, `std::cerr` in the standard error output stream.
`std::cout` can only be used in _CLI_ mode php. For creating web applications, this method does not work.
The simplest method to solve the issue is to write a wrapper around the `zend_write`:

``` c
void echo(std::string str) {
    zend_write( str.c_str(), str.size());
}
```

I just checked. This function works!
Basically, in this form it is enough, but we can go ahead and implement something like this:

``` c
Php::echo << "PI equals " << 3.14;
```
